Maple2Storage/Scripts/Functions/calcEnchantValues.lua

```diff
@@ -31,18 +31,17 @@ function calcEnchantIngredients(enchantLevel, rarity, itemSlot, itemLevel)
     local ingredient2Tag = 'CrystalPiece'
     local ingredient3Tag = 'ChaosOnix'
     local catalystCost = 0
-    local nextEnchantLevel = enchantLevel + 1
-    local ingredient1Cost = 100 * nextEnchantLevel -- temp value
-    local ingredient2Cost = 100 * nextEnchantLevel -- temp value
-    local ingredient3Cost = 100 * nextEnchantLevel -- temp value
+    local ingredient1Cost = 100 * enchantLevel -- temp value
+    local ingredient2Cost = 100 * enchantLevel -- temp value
+    local ingredient3Cost = 100 * enchantLevel -- temp value
     -- TODO: // figure out ingredient cost calculation
-    if nextEnchantLevel > 10 then
+    if enchantLevel > 10 then
         catalystCost = 1
     end
     return catalystCost, ingredient1Tag, ingredient1Cost, ingredient2Tag, ingredient2Cost, ingredient3Tag, ingredient3Cost
 end
 
-EnchantingBonusFactor = { 0.02, 0.02, 0.03, 0.03, 0.04, 0.05, 0.06, 0.07, 0.08, 0.1, 0.14, 0.2, 0.28, 0.38, 0.5 }
+EnchantingBonusFactor = { 0.02, 0.04, 0.07, 0.1, 0.14, 0.19, 0.25, 0.32, 0.4, 0.5, 0.64, 0.84, 1.12, 1.5, 2 }
```

Comment:
Enchanting normally is one level at a time. If a user uses a enchant scroll, for example, +10, if I kept the current formula, I'd have to either create a loop to add all the values together or just change the values in `EnchantingBonusFactor` to reference the value of the specific level, instead of just the difference between current level and past level.
It helps keep using the same formula for all possible enchanting.